plugin/evm/block_builder.go

```diff
@@ -36,9 +40,17 @@ type blockBuilder struct {
 	ctx         *snow.Context
 	chainConfig *params.ChainConfig
 
+	config Config
```

Comment:
can we create a separate block builder config that only includes the necessary parameters for the block builder instead of passing through the VM's whole config?